Assets/Mirror/Runtime/NetworkConnectionToClient.cs

```diff
@@ -16,6 +16,17 @@ public NetworkConnectionToClient(int networkConnectionId) : base(networkConnecti
         // the client. they would be detected as a message. send messages instead.
         readonly List<int> singleConnectionId = new List<int> { -1 };
 
+        /// <summary>
+        /// Number of seconds of no messages from client after which server will auto-disconnect
+        /// </summary>
+        public float serverIdleTimeout = 30f;
```

Comment:
Maybe this should be a field on the transport, so that users can change it.

Reply:
The problem is the transports are not telling the server the client is dead.